src/components/Integrations/Shopify/Shopify.css

```diff
@@ -0,0 +1,8 @@
+img {
+    width: 10%;
```

Comment:
Why is it a percentaje? is it fine apply this style for all `img`s?

Reply:
All images elements inside shopify component only. But I better add a class..

Reply:
The idea of having css inside a component is that it applies only to this component.